src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -256,10 +267,31 @@ export const CustomerQuotaScreen: FunctionComponent<CustomerQuotaProps> = ({
     clearError,
     error,
     expireSession,
-    showAlert
+    showAlert,
+    quotaState,
+    quotaError
   ]);
 
-  return cartState === "FETCHING_QUOTA" ? (
+  useEffect(() => {
+    /**
+     * Update quota after checkout
+     */
+    if (cartState === "PURCHASED" && !updateAfterPurchased) {
+      updateQuota();
+      setUpdateAfterPurchased(true);
+    }
+  }, [cartState, updateQuota, updateAfterPurchased]);
+
+  useEffect(() => {
+    /**
+     * Initialise available cart items when quota or product changes
+     */
+    if (quotaResponse?.remainingQuota) {
+      initCartWithQuota(quotaResponse.remainingQuota);
+    }
+  }, [quotaResponse, initCartWithQuota, products]);
```

Comment:
What are your thoughts if we pass it as a prop into `useCart` instead? Something like `initialCartQuota`, then inside the hook, an `useEffect` merges the `initialCartQuota` with the existing cart whenever it changes.

Reply:
I agree with this, this will inform developers that there is a dependency on `useQuota` as well.
This has been implemented.